test/e2e/data/cni/calico/calico.yaml

```diff
@@ -0,0 +1,792 @@
+---
+# Source: calico/templates/calico-config.yaml
```

Comment:
Should this comment include a version so that it's easier to track when we should update?